appservice/src/connectToGitHub.ts

```diff
@@ -64,11 +67,13 @@ export async function connectToGitHub(node: AzureTreeItem, client: SiteClient, c
     };
 
     try {
-        const connectingToGithub: string = localize('ConnectingToGithub', '"{0}" is being connected to the GitHub repo. This may take several minutes...', client.fullName);
-        const connectedToGithub: string = localize('ConnectedToGithub', '"{0}" has been connected to the GitHub repo.', client.fullName);
+        const connectingToGithub: string = localize('ConnectingToGithub', '"{0}" is being connected to the GitHub repo... Check output channel for status', client.fullName);
+        const connectedToGithub: string = localize('ConnectedToGithub', 'The GitHub repo is now connected and is being deployed to "{0}" .', client.fullName);
```

Comment:
Why "is being deployed"? Isn't the deploy already finished?

Reply:
Well, it's connected and deployed, but still building according to the status codes. The deployment doesn't become active until after it's finished building so I thought it'd be misleading to say that it's deployed.